reconcile/gitlab_members.py

```diff
@@ -127,11 +154,11 @@ def subtract_states(
 ) -> list[Diff]:
     """Return diff objects for items in from_state but not in subtract_state."""
     result = []
-    for f_group, f_users in from_state.items():
-        s_group = subtract_state[f_group]
-        for f_user in f_users:
+    for f_group_name, f_gitlab_group in from_state.items():
+        s_group = subtract_state[f_group_name]
+        for f_user in f_gitlab_group.members:
             found = False
-            for s_user in s_group:
+            for s_user in s_group.members:
```

Comment:
to avoid iterate all members to find match, can change `members` to be a dict, key is name, value is member object, and can be used to simplify `check_access` too